pkg/k8s/report/report.go

```diff
@@ -90,8 +90,13 @@ func (r Report) consolidate() ConsolidatedReport {
 	for _, m := range r.Resources {
 		if vulnerabilitiesOrSecretResource(m) {
 			vulnerabilities = append(vulnerabilities, m)
-		} else {
+		}
+		if misconfigsResource(m) {
+			res, ok := index[m.fullname()]
 			index[m.fullname()] = m
+			if ok {
+				index[m.fullname()].Results[0].Misconfigurations = append(index[m.fullname()].Results[0].Misconfigurations, res.Results[0].Misconfigurations...)
+			}
```

Comment:
Could you add a test in `report_test.go` to cover this case?

Reply:
good catch! thanks! 
done https://github.com/aquasecurity/trivy/pull/7797/commits/4bc54345a07542fc3ada52f8df87b3837c8af9f5